main.py

```diff
@@ -0,0 +1,482 @@
+import base64
+import hashlib
+import os
+import os.path as osp
+import re
+import shutil
+import subprocess
+import tempfile
+
+from urllib.parse import quote, unquote
+from urllib.request import urlopen
+
+import sublime
+import sublime_plugin
+
+from sublime import View
+
+from .utils.get_image_size import get_image_size, UnknownImageFormat
+
+
+TEMPLATE = """
```

Comment:
![image](https://user-images.githubusercontent.com/33235928/58763850-e3feec00-8560-11e9-9488-09d90a0c816f.png)

can you play with the style so that small images gets centered in the popup ?

Reply:
Let's open a separate ticket with this. Though I don't feel that it should be centered. Any reason why you think centered would be better for smaller images?

Reply:
If you don't see a necessity for this, drop it, I just thought it'd look better.